docs/nomad_simulations.md

```diff
@@ -1 +1,499 @@
+
+??? success "Solution 2.4"
+    We can import and create the class `SelfConsistency`, assign the specified quantities, and append it under `dft`:
+    ```python
+    from nomad_simulations.schema_packages.numerical_settings import SelfConsistency
+    scf = SelfConsistency(threshold_change=1e-3, threshold_change_unit='joule')
+    dft.numerical_settings.append(scf)
+    ```
+
+    In order to see if `scf` is showing directly under `simulation`, we can:
+    ```python
+    simulation.model_method[0].numerical_settings
+    ```
+    which indeed returns:
+    ```sh
+    [SelfConsistency:SelfConsistency(name, threshold_change, threshold_change_unit)]
+    ```
+
+    In order to go from `scf` to the `dft.jacobs_ladder` information, we need to go one level up with respect to `scf`. In order to do this, we can use the method `m_parent`:
+    ```python
+    scf.m_parent.jacobs_ladder
+    ```
+    which will return:
+    ```sh
+    'LDA'
+    ```
+
+## `ModelSystem` {#modelsystem}
+
+The `ModelSystem` section is an input section which contains all the information about the geometrical space quantities (positions, lattice vectors, volumes, etc) of the simulated system. This section contains various quantities and sub-sections which aim to describe the system in the most complete way and in a variety of cases, from unit cells of crystals and molecules up to microstructures. In order to handle this _hierarchical_ structure, `ModelSystem` is nested over itself, i.e., a `ModelSystem` can be composed of sub-systems, which at the same time could be composed of smaller sub-systems, and so on. This is done thanks to the (proxy) sub-section attribute called `model_system`.
+
+The `Cell` sub-section is an important section which contains information of the simulated cell. It does not necessarily contain the information of the elements sitting in each `positions`, as this is responsability of a more specialized section, the `AtomicCell`. This section contains information about the atoms conforming the material in the [`AtomsState`](#atomsstate) section. The `Symmetry` sub-section contains information about the symmetry of the system, and the `ChemicalFormula` sub-section about the strings that allow the system to be identified in a specific format of the chemical formulas (IUPAC, Hill, etc).
+
+The detailed relationship tree is:
+
+<div class="click-zoom">
+    <label>
+        <input type="checkbox">
+        <img src="../assets/part2-nomad-simulations/model_system.png" alt="ModelSystem quantities and functions UML diagram." width="80%" title="Click to zoom in">
+    </label>
+</div>
+
+The abstract section `Entity` is defined in the [Basic Formal Ontology (BFO)](https://basic-formal-ontology.org/) as a continuant entity, and we use it to abstract our `ModelSystem` using the intermediate base section `System`. This base section, `System`, is also used by the experimental data models.
```

Comment:
I feel like this sentence may be a bit confusing for a beginner. Can you try to simplify/clarify the point?

Reply:
I tried it out, but I am not 100% convinced